### PR TITLE
added five questions

### PR DESCRIPTION
**Pull request by amitkroutthedev**
added questions in JS from id 11 to id 16.

### CODE REVIEW

**Comment by atiqueakhtar** [comment]
Hi @amitkroutthedev don't we need to add anything in answers.json?? Just curious

**Comment by amitkroutthedev** [comment]
I was also little confused because there was no ID in answer.json as there was in question.json

**Comment by amitkroutthedev** [comment]
If there is some error please suggest so that I can change and give a proper pull request

File: Languages/JS/questions.json [hunk 1]
```diff
@@ -108,6 +108,51 @@
 		"trueOrFalse": false,
 		"shortAns": "It is an object, which object depends upon where the code is being invoked.",
 		"tag": "JavaScript"
+	},
+	{
+		"id": 12,
```
Comment (Tc2r1) [inline Languages/JS/questions.json:113]: This question and shortAns are way too obvious. Please phrase the shortAns as "It is ..." and please dont use the wording of the question itself in the answer, that is a dead give away.

please don't use the word in the shortAns.

"what does a cat eat?"
a) oranges
b) tacos
c) a cat eats mice
d) trees.

making the answer so obvious defeats the point of the app.

File: Languages/JS/questions.json [hunk 2]
```diff
@@ -108,6 +108,51 @@
...
+		"details": "IIFE is a function in Javascript which immediately invoked and executed as soon as it is defined. Variables declared within the IIFE cannot be accessed by the outside world and this way you can avoid the global scope from getting polluted. So the primary reason to use IIFE is to immediately execute the code and obtain data privacy.",
+		"questionType": "multi",
+		"trueOrFalse": false,
+		"shortAns": "IIFE is a function in Javascript which immediately invoked and executed as soon as it is defined.",
+		"tag": "JavaScript"
+	},
+	{
+		"id": 13,
+		"question": "What is Closure?",
+		"details": "A closure is simply a function inside another function that has access to the outer function variable. Now, this definition sound pretty much straightforward but the real magic is created with the scope. The inner function can access the variable defined in its scope, in the scope of its parent function, and the global variables.",
```
Comment (Tc2r1) [inline Languages/JS/questions.json:124]: Please decrease each of your "details" by one sentence.

**Review by Tc2r1 — CHANGES_REQUESTED** [review]

**Review by Tc2r1 — APPROVED** [review]

**Pull request merged**